Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -163,3 +163,27 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
         self.isBeta = isBeta
     }
 }
+
+extension Array where Array.Element == AvailabilityRenderItem {
+    /// Determines whether all platforms in the array are in beta.
```

Comment:
This is a verb phrase ("determines") but properties is more commonly described by nouns. 

We could either change this to a function (`func allAreInBeta() -> Bool`) to go better with the verb phrase or change this to a noun phrase. For example:
```suggestion
    /// A Boolean value that determines whether all platforms in the array are in beta.
```

Reply:
That said, with a simple and clean name like `allAreInBeta` I don't know if this needs any documentation at all as an _internal_ API.